### DISCUSSION TITLE
<title>A task that I need help in

### DISCUSSION THREAD

**Discussion by Zilla908** [community/community#112538] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello anybody know what I'm supposed to do here, the 3. assingment is where I'm stuck. As shown in the screenshots I can't start a pull request so where do I start a pull request for the commit
![Screenshot_13-3-2024_101727_github com](https://github.com/community/community/assets/159877215/44a23aca-f630-4c48-a736-23254381bc26)
![Screenshot_13-3-2024_10162_github com](https://github.com/community/community/assets/159877215/24687e27-65de-4c1f-895f-0b8435c4ffb2)
![Screenshot_13-3-2024_101457_github com](https://github.com/community/community/assets/159877215/b41e5b60-5494-49fa-8454-aa09ffdeab64)

**Comment by HafizMustaqim** [comment]
have you checked your stashed?

**Reply by Zilla908** [reply]
where would this be ?

**Reply by Zilla908** [reply]
I think I made a mistake on the task so I should start again

**Comment by DNin01** [comment]
It appears that you may have committed directly to `main` instead of `fix-sidebar`.

Always make sure you have checked out the branch you want to change before committing. Don't worry, this mistake happens to the best of us!

**Comment by moozunch** [comment]
In order to be able to pull request is you have to commit in the other branch other than main, make sure where branch you commit to before commiting

**Comment by muskaanrajput14** [comment]
you are not committing, commit something

**Comment by Zilla908** [comment]
I have succeeded on the task
